docs/user-guide/composer.md

```diff
@@ -1,3 +1,45 @@
-# Composer
 
-TODO
+ This guide will walk you through the process of using our email composition tool effectively.
+ ###  Creating a New Email in web
+1. Click on the "New Email" button.
+2. A new email composer will appear.
+3. From: By default it will display your default identity. When you click on `Edit` icon (3.1)- a drop-down list of your identities will be shown and you can select one of them
+4. Fill in the recipient's email address in the "To" field.
+5. Enter a subject for your email in the "Subject" field.
+6. Adding Recipients
+You can add multiple recipients by separating their email addresses with commas. Use the "CC" (Carbon Copy) and "BCC" (Blind Carbon Copy) fields for additional recipients as needed.
+- (6.1)`CC`: Carbon Copy:  Anyone in this field will see the other recipients of the message.
+- (6.2) `BCC`: Blind Carbon Copy: The recipients won't know that you added anyone to 'Bcc'.
+![image](https://github.com/linagora/tmail-flutter/assets/68209176/7eb71fd0-0f2c-4b9b-9bbe-e5e55e36f596)
```

Comment:
Please use the new composer -> https://oncommit-tmail.linagora.com for the screenshots.